### PR TITLE
Added information about specifying version when installing snaps

### PR DESCRIPTION
**Pull request by ritave**
* Added Awesome Metamask Snaps link

### CODE REVIEW

File: docs/guide/snaps-rpc-api.md [hunk 1]
```diff
@@ -226,6 +235,8 @@ interface WalletInstallSnapsResult {
 This method attempts to install the requested snaps, if they are permitted.
 If the installation of any snap fails, its object value on the result will contain an `error` property with the error that caused the installation to fail.
 
+Optionally you can specify a [SemVer range](https://www.npmjs.com/package/semver) of a Snap to be installed. Metamask will try to install a version that is max-satisfying of all the possible versions within that range. Should a Snap be already installed with a version that isn't in the requested range, Metamask will try to upgrade the installed Snap to satisfy the range, or an error returned if an upgrade is not possible or denied by the user.
```
Comment (mcmire) [inline docs/guide/snaps-rpc-api.md:238]: Is it our convention to capitalize "snap" or leave it uncapitalized?

Also Metamask -> MetaMask

File: docs/guide/snaps.md [hunk 2]
```diff
@@ -76,8 +76,7 @@ Once you've connected to Flask, smash that `Send Hello` button, and watch the sn
...
 ## Where to Go from Here
 
-If you're interested in developing your own snap, your next destination is probably [the snaps development guide](./snaps-development-guide.html).
-
-You may also wish to check out the first functional key management snap ever made, [the Filecoin Snap, or `filsnap`](https://github.com/Chainsafe/filsnap/).
-
-Currently, snaps are discovered through word-of-mouth. As the system matures, MetaMask will provide means for both developers and users to discover snaps.
+- If you're interested in developing your own snap, your next destination is probably [the snaps development guide](./snaps-development-guide.html).
+- You may also wish to check out the first functional key management snap ever made, [the Filecoin Snap, or `filsnap`](https://github.com/Chainsafe/filsnap/).
+- Checkout the [Awesome list of Snaps related resources](https://github.com/piotr-roslaniec/awesome-metamask-snaps)
```
Comment (mcmire) [inline docs/guide/snaps.md:81]: Checkout -> Check out

And add a period at the end of this.

File: docs/guide/snaps-rpc-api.md (hunk 1 above)
Comment (mcmire) [reply docs/guide/snaps-rpc-api.md:238]: Also, "Metamask will try to install a version that is max-satisfying of all the possible versions within that range" sounds a bit strange. Maybe "MetaMask will try to install a version that satisfies the maximum version specified by the range?"

**Review by mcmire — COMMENTED** [review]
Looks good, just a few nits.

File: docs/guide/snaps-rpc-api.md (hunk 1 above)
Comment (rekmarks) [reply docs/guide/snaps-rpc-api.md:238]: Regarding the capitalization of "snap"/"snaps", we haven't established any conventions, but I've been doing this:
- "Snaps" refers to the feature / system as a whole.
- A "snap" / several "snaps" refers to individual snap(s) developed by someone.

**Review by rekmarks — APPROVED** [review]

**Pull request merged**